sklearn/model_selection/_search.py

```diff
@@ -371,7 +371,7 @@ class BaseSearchCV(six.with_metaclass(ABCMeta, BaseEstimator,
     def __init__(self, estimator, scoring=None,
                  fit_params=None, n_jobs=1, iid=True,
                  refit=True, cv=None, verbose=0, pre_dispatch='2*n_jobs',
-                 error_score='raise'):
+                 error_score='raise', return_train_score=False):
```

Comment:
I guess making it optional is helpful if you have a very large training set on which evaluation takes a long time because of an very expensive evaluation procedure. But I think we should default to ``True`` both for convenience and discoverability.